src/Pim/Bundle/DataGridBundle/EventListener/AddParametersToVariantProductGridListener.php

```diff
@@ -1,70 +0,0 @@
-<?php
```

Comment:
The listener itself is still here as we need here. Only the class is removed as it does nothing.